lang/de.json

```diff
@@ -1,13 +1,19 @@
 {
     "IPv6Usage": {
         "PluginDescription": "IPv6 Usage: Dieses Plugin zeigt die Anzahl der Besuche durch IPv4 und IPv6",
-        "ProtocolUsageEvolution": "IP Protokoll Entwicklung",
-        "WidgetProtocolDescription": "IP Protokoll nach Version",
+        "ProtocolUsageEvolution": "IP-Protokoll-Entwicklung",
+        "WidgetDescription": "",
+        "WidgetProtocolDescription": "IP-Protokoll nach Version",
         "IPv4": "IPv4",
         "IPv6": "IPv6",
-        "IPv4_rate": "Anzahl von IPv4 Besuchern",
-        "IPv6_rate": "Anzahl von IPv6 Besuchern",
-        "IPProtocol": "IP Protokoll",
-        "Evolution": "Evolution"
+        "Teredo": "Teredo-Tunnel",
+        "Tun6to4": "6to4-Tunnel",
+        "IPv4_rate": "Anzahl von IPv4-Besuchern",
```

Comment:
How about just "Anzahl IPv4-Besucher" oder "Anzahl Besucher über IPv4"?  Same for IPv6 of course.

Reply:
You're right, that's more elegant. I'll change that, and then open a new pull request.